packages/core-data/src/queried-data/reducer.js

```diff
@@ -222,19 +222,22 @@ const receiveQueries = compose( [
 	// Queries shape is shared, but keyed by query `stableKey` part. Original
 	// reducer tracks only a single query object.
 	onSubKey( 'stableKey' ),
-] )( ( state = null, action ) => {
+] )( ( state = {}, action ) => {
```

Comment:
Do you think there will be nuances by changing the default value here?

Reply:
no, it's fine, it's just about ensuring the right shape of state (since we modified it)